packages/odyssey-react-mui/src/DataTable/DataTable.tsx

```diff
@@ -382,7 +389,7 @@ const DataTable = ({
     useState<MRT_VisibilityState>();
   const [rowDensity, setRowDensity] =
     useState<MRT_DensityState>(initialDensity);
-  const [rowSelection, setRowSelection] = useState<MRT_RowSelectionState>({});
+  const [rowSelection, setRowSelection] = useState<MRT_RowSelectionState>(initialRowSelection || {});
```

Comment:
I don't think this is gonna work the way we want.

If the table never gets unmounted, then the passed-in row selection will only work on the first render.

For instance, what if the pagination changes and now we're on rows 11-20? They won't be checked.

What if we refresh the list of items while the user is on the table, and a new item shows up that's supposed to be checked now?